specutils/tests/test_loaders.py

```diff
@@ -1273,3 +1273,64 @@ def test_galah_guess(remote_data_path):
         assert isinstance(spectra[1].uncertainty, StdDevUncertainty)
         assert spectra[1].meta.get("label") is not None
         assert spectra[1].meta.get("header") is not None
+
+
+# We cannot use remote_access directly in the MIRI MRS tests, because
+# the test functions are called once for every file in the remote access
+# list, but we need to feed the entire set to the functions at once. We
+# store the file names in a list and use a dummy test method to load the
+# list via the remote_access machinery.
+#
+# The zenodo record contains the entire set of 12 _x1d files corresponding
+# to this data set. We use just a few files for the current tests. Maybe
+# more files will be needed when more functionality is added later on.
+filename_storage = []
+@remote_access([
+    {'id': '5082863', 'filename': 'combine_dithers_all_exposures_ch1-long_x1d.fits'},
+    {'id': '5082863', 'filename': 'combine_dithers_all_exposures_ch1-medium_x1d.fits'},
+    {'id': '5082863', 'filename': 'combine_dithers_all_exposures_ch1-short_x1d.fits'},
+])
+def test_loaddata_miri_mrs(remote_data_path):
+    filename_storage.append(remote_data_path)
+
+
+# loading from a list of file names
+@pytest.mark.remote_data
+def test_spectrum_list_names_miri_mrs():
+
+    # format is explicitly set
+    specs = SpectrumList.read(filename_storage, format="JWST x1d MIRI MRS")
+
+    assert len(specs) == 3
+    for spec in specs:
+        assert isinstance(spec, Spectrum1D)
+        assert spec.spectral_axis.unit == u.Unit("um")
+
+    # auto-detect format
```

Comment:
Good to keep that in mind. I will leave it as is for now though, since we don't know yet how this is going to evolve. Maybe more tests will be required anyway.

Reply:
For a given file there would normally only be the two cases of specifying `format` explicitly or not, seems unlikely more options could be coming in there.